index.html

```diff
@@ -65,6 +65,8 @@ <h2 class="categories">
     </div>
     </main>
     <br>
+    <!-- Add iframe -->
+    <iframe src="http://127.0.0.1:5500/COMP-333-HW1/reviews.html#" frameborder="0" width="100%" height="400"></iframe>
```

Comment:
`src="http://127.0.0.1:5500/COMP-333-HW1/reviews.html#"` -> `"src=/reviews.html#"`
I think this can just be src="/reviews.html#". Including the local server in the path stops the page from loading for other people.